homeassistant/components/google/calendar.py

```diff
@@ -367,6 +368,13 @@ def event(self) -> CalendarEvent | None:
         return event
 
     def _event_filter(self, event: Event) -> bool:
+        if any(
```

Comment:
I do not think this is a reliable signal because of different ways email addresses are formatted and multiple accounts.  Looking at https://developers.google.com/calendar/api/v3/reference/events the attendee has a `self` so maybe we can use that? But one challenge is i think we strip it https://github.com/allenporter/gcal_sync/blob/d367b9b96fe4eb270e8fa8fea7e1be62080c5743/gcal_sync/model.py#L115 because it conflicts with the python variable.  Maybe we can add that back and use it here...

Reply:
Ah, that would do it! I had tried implementing this using the `self` attribute but couldn't figure out what was stripping it out. PR for `gcal_sync` here: https://github.com/allenporter/gcal_sync/pull/600 ❤️

Reply:
Bumping library in https://github.com/home-assistant/core/pull/128949

Reply:
I think we're all set here! Moved out of draft.